dvc/remote/ssh/__init__.py

```diff
@@ -38,15 +39,27 @@ def __init__(self, repo, config):
 
         parsed = urlparse(self.url)
         self.host = parsed.hostname
+
+        user_config_file = os.path.expanduser("~/.ssh/config")
+        user_ssh_config = dict()
+        if self.host:
```

Comment:
Could you please elaborate why `if self.host` is needed?

Reply:
@efiop it's fix only for tests :(

dependency test
https://github.com/iterative/dvc/blob/master/tests/func/test_dependency.py#L20
create remotes with empty configs

here the actual place where empty remote config created
https://github.com/iterative/dvc/blob/master/dvc/output/base.py#L74

also we have weird place
https://github.com/iterative/dvc/blob/master/dvc/remote/ssh/__init__.py#L37

```python
self.url = config.get(Config.SECTION_REMOTE_URL, "ssh://")
```
obviously ssh will not work with such default url and it should be error.
I think it also fix for dependency test, but right now I don't know how make it beautiful. But I will research it as a separate task.